docs/contact.md

```diff
@@ -1,2 +1,3 @@
-#Contact us
-Mail us at [ultra-oncall@flipkart.com](mailto:ultra-oncall@flipkart.com)
+# Contact Us
+
+For any other queries, please drop an email to us at **ultra-oncall@flipkart.com**.
```

Comment:
Please add a link (mailto:ultra-oncall@flipkart.com)

Reply:
Added the link.